src/libraries/Common/src/System/Drawing/KnownColorTable.cs

```diff
@@ -124,6 +124,7 @@ internal static class KnownColorTable
             0xFFDDA0DD,     // Plum
             0xFFB0E0E6,     // PowderBlue
             0xFF800080,     // Purple
+            0xFF663399,     // RebeccaPurple
```

Comment:
This is in the wrong location, it must match the order of the enum

Reply:
ok

Reply:
i moved it to the bottom. it should be fixed now